src/main/java/digital/slovensko/autogram/ui/gui/SigningDialogController.java

```diff
@@ -54,6 +56,13 @@ public SigningDialogController(Visualization visualization, Autogram autogram, G
     }
 
     public void initialize() {
+        TokenDriver driver = UserSettings.load().getDriver();
+        if (driver != null) {
```

Comment:
To volanie request password pri každom podpise je ok? Mne to nejako zblbne aj s OP po treťom podpise, že sa nepodarilo načítať kartu. A musím až reštartovať autogram, aby to začalo zase fungovať. S gemalto sa mi to pokazí takmer hneď - asi pri druhom podpise.